types/mui-datatables/index.d.ts

```diff
@@ -126,24 +132,30 @@ export interface MUIDataTableColumnState extends MUIDataTableColumnOptions {
 }
 
 export interface MUIDataTableColumnOptions {
-    customBodyRender?: (value: any, tableMeta: MUIDataTableMeta, updateValue: (value: string) => void) => string | React.ReactNode;
-    customHeadRender?: (columnMeta: MUIDataTableCustomHeadRenderer, updateDirection: (params: any) => any) => string | React.ReactNode;
-    customFilterListRender?: (value: any) => string;
+    customBodyRender?: (
+        value: any,
+        tableMeta: MUIDataTableMeta,
+        updateValue: (value: string) => void
+    ) => string | React.ReactNode;
     customFilterListOptions?: MUIDataTableCustomFilterListOptions;
+    customFilterListRender?: (value: any) => string;
+    customHeadRender?: (
+        columnMeta: MUIDataTableCustomHeadRenderer,
+        updateDirection: (params: any) => any
+    ) => string | React.ReactNode;
     display?: 'true' | 'false' | 'excluded';
     download?: boolean;
     empty?: boolean;
     filter?: boolean;
-    filterType?: FilterType;
     filterList?: string[];
     filterOptions?: MUIDataTableFilterOptions;
+    filterType?: FilterType;
     hint?: string;
     print?: boolean;
     searchable?: boolean;
     setCellHeaderProps?: (columnMeta: MUIDataTableCustomHeadRenderer) => object;
     setCellProps?: (cellValue: string, rowIndex: number, columnIndex: number) => object;
     sort?: boolean;
-    sortDirection?: 'asc' | 'desc' | 'none';
```

Comment:
can you left this as is and mark wiht proper JSDOC? What is better for users?
```
/** @deprecated */
```
this will allow users to recognize the fact that code changed:
https://github.com/gregnb/mui-datatables/blob/49c18a4ccffeccc684c6091cae968af441d2568b/src/MUIDataTable.js#L510